test/typings/common.ts

```diff
@@ -0,0 +1,22 @@
+
+export type RootState = {
+	items: {[key: string]: {
```

Comment:
Can you convert tabs to spaces to match the other files? It's a small thing, but it'll help avoid whitespace changes on future diffs.